docs/guides/index.md

```diff
@@ -527,15 +527,16 @@ Events][animatingonevents] below.
 
 In JavaScript, we grab the box with [`querySelector`][queryselector], use
 [`addEventListener`][addeventlistener], and then [`setAttribute`][setattribute]
-to make the box grow its scale when its hovered over. Note that A-Frame adds
+to make the box grow its scale when its hovered over. The code must appear after
```

Comment:
only one mention of scripts in getting started tutorial before that point.

> We include A-Frame as a script tag in the <head>, pointing to the A-Frame build hosted on a CDN. This has to be included before <a-scene> because A-Frame registers custom HTML elements which must be defined before <a-scene> is attached or else <a-scene> will do nothing.

then from this example

>One way to manually handle the cursor events is to add an event listener with JavaScript just like we would with a normal DOM element. If you aren’t comfortable with JavaScript, you may skip to Animating on Events below.

> In JavaScript, we grab the box with querySelector, use addEventListener, and then setAttribute to make the box grow its scale when its hovered over. Note that A-Frame adds features to setAttribute to work with multi-property components. We can pass a full {x, y, z} object as the second argument.